sql-aggregates/movie-maniacs.sql

```diff
@@ -0,0 +1,6 @@
+SELECT "c"."firstName",
+      "c"."lastName",
+      SUM("amount")
+FROM "payments"
+JOIN "customers" as "c" using ("customerId")
+GROUP BY "customerId"
```

Comment:
Don't forget to make sure they're in order:

---

![image](https://user-images.githubusercontent.com/16143771/144960795-7e115d13-237e-44b6-a48b-3c23e5eb0cd2.png)